src/Components/Announcement.vue

```diff
@@ -41,6 +41,10 @@
 
 					<template v-if="isAdmin">
 						·
+						<template v-if="isScheduled">
```

Comment:
This is a bit problematic is seems:
![grafik](https://github.com/user-attachments/assets/4cf16ed9-3ae5-4818-bc9f-07c5d265669b)

I would say that the schedule time could replace the posting time

Reply:
![Bildschirmfoto vom 2024-07-29 10-11-59](https://github.com/user-attachments/assets/f861060e-8fa0-469f-82e6-077f452a5cbc)

Proposed change, I think this looks cleaner now, good catch!